clients/rust/src/generated/instructions/create_tree_config.rs

```diff
@@ -0,0 +1,555 @@
+//! This code was AUTOGENERATED using the kinobi library.
+//! Please DO NOT EDIT THIS FILE, instead use visitors
+//! to add features, then rerun kinobi to update it.
+//!
+//! [https://github.com/metaplex-foundation/kinobi]
+//!
+
+use borsh::BorshDeserialize;
+use borsh::BorshSerialize;
+
+/// Accounts.
+pub struct CreateTreeConfig {
+    pub tree_config: solana_program::pubkey::Pubkey,
+
+    pub merkle_tree: solana_program::pubkey::Pubkey,
+
+    pub payer: solana_program::pubkey::Pubkey,
+
+    pub tree_creator: solana_program::pubkey::Pubkey,
+
+    pub log_wrapper: solana_program::pubkey::Pubkey,
+
+    pub compression_program: solana_program::pubkey::Pubkey,
+
+    pub system_program: solana_program::pubkey::Pubkey,
+}
+
+impl CreateTreeConfig {
+    pub fn instruction(
+        &self,
+        args: CreateTreeConfigInstructionArgs,
+    ) -> solana_program::instruction::Instruction {
+        self.instruction_with_remaining_accounts(args, &[])
+    }
+    #[allow(clippy::vec_init_then_push)]
+    pub fn instruction_with_remaining_accounts(
```

Comment:
Do we need `instruction_with_remaining_accounts`?  I don't think there's any reason to add remaining accounts to `create_tree`.  Same is true for other instructions such as `set_decompressable_state`.

Reply:
This is auto-generated for all instructions.